client/a8c-for-agencies/sections/referrals/primary/referrals-overview/index.tsx

```diff
@@ -60,6 +62,10 @@ export default function ReferralsOverview( {
 			: translate( 'Referrals' );
 
 	const { data: tipaltiData, isFetching } = useGetTipaltiPayee();
+	const accountStatus = getAccountStatus( tipaltiData, translate );
+	const referralsAvailable = accountStatus?.statusType === 'success';
+	const actionRequiredNotice = ! isFetching && accountStatus?.statusType === 'warning';
```

Comment:
It is better to check if `IsPayable` is false here & show this notice only when we have referrals since this state will be handled in the empty state.

Reply:
I was thinking of showing the banner only when a user has a referral, i.e not show it in the empty state since it will be handled in the empty view, but show it when the table view is shown as per the design here:

It says:

```
If we can, in the situation where it changed from confirmed to some sort of error. We should show a banner. We should also show a notification in the sidebar entry. This will make sure the user goes to Payment settings.
```

pfSQfS-1w-p2


<img width="873" alt="Screenshot 2024-07-12 at 11 36 37 AM" src="https://github.com/user-attachments/assets/76d6cd0e-d208-469d-b670-5b35bf64ff2b">

What do you think?